templates/result-email-template.html

```diff
@@ -72,13 +98,23 @@ <h2>General Information</h2>
             <td>{{ log_link }}</td>
         </tr>
         <tr>
-            <th style="color: blue">Total suite duration</th>
-            <td>{{ suite_run_time }}</td>
+            <th style="color: blue">Invoked by User</th>
+            <td>{{ trigger_user }}</td>
+        </tr>
+    </table>
+    <h3>Suite duration</h3>
+    <table class="half-table">
+        <tr>
+            <th style="color: blue">Start Time</th>
+            <td>{{ suite_run_time.start }}</td>
         </tr>
+        <tr>
+            <th style="color: blue">End Time</th>
+            <td>{{ suite_run_time.end }}</td>
         </tr>
         <tr>
-            <th style="color: blue">Invoked by User</th>
-            <td>{{ trigger_user }}</td>
+            <th style="color: blue">Execution Duration</th>
+            <td>{{ suite_run_time.total }}</td>
```

Comment:
one of the example of `total`  is  `4.0 minutes 0.2220790000000079 seconds` . I this is not ideal for readability. can you reduce the decimal to points ?

Reply:
Yeah, Sorry, I have changed it since then and the mailw as not with final rendering. I will post the log with all the changes in place.